ImportExport/Strategy/StrategyRelationsTrait.php

```diff
@@ -39,13 +39,6 @@ protected function updateRelations($entity, array $itemData = null)
                 $isFullRelation = $this->fieldHelper->getConfigValue($entityName, $fieldName, 'full', false);
                 $isPersistRelation = $this->databaseHelper->isCascadePersist($entityName, $fieldName);
 
-                $searchContext = $this->generateSearchContextForRelationsUpdate(
```

Comment:
execution of this method causes populating lazy load proxy for all relations - even if the relation entity does not exist. Moving the execution into the if statement AND even deeper into the condition when only the relation exists speeds up the execution very much. It will not be noticible for schemas with ~200 attributes, although for bigger product attributes counts with big set of relations - this is a massive improvement.